docs/choreolib/basic-usage.md

```diff
@@ -1,44 +1,57 @@
-# Basic Usage
+# Getting Started
 
+## Auto Factory API
+!!! tip
+      This API is recommended for most teams as it provides a more user-friendly, batteries included experience.
+The `AutoFactory` class provides a high level API that simplifies the creation of complex autonomous routines in your robot code. Read [the documentation page](./auto-factory.md) for more information.
+
+## Raw Trajectory API
 Teams that want to use their own path following structure can load trajectories directly with the following code.
 
-``` { .java .select }
-import java.util.Optional;
-
-import choreo.Choreo;
-import choreo.trajectory.ChoreoTrajectory;
-import choreo.trajectory.SwerveSample;
-
-// Loading a trajectory from a file, returns an optional if the file does not exist or is invalid
-var trajectory = Choreo.loadTrajectory("myTrajectory");
-if (trajectory.isPresent()) {
-    // Do something with the trajectory
-    drive.followTrajectory(trajectory.get()).schedule();
-} else {
-    // If the trajectory is not found, ChoreoLib already prints to DriverStation
-}
-```
-``` { .py .select }
-import choreo
-
-# Loading a trajectory from a file, throwing ValueError if the file does not exist or is invalid
-try:
-    trajectory = choreo.load_swerve_trajectory("myTrajectory")
-
-    # Do something with the trajectory
-    drive.followTrajectory(trajectory).schedule()
-except ValueError:
-    # If the trajectory is not found, ChoreoLib already prints to DriverStation
-    pass
-```
-``` { .cpp .select }
-#include <choreo/Choreo.h>
-
-// Loading a trajectory from a file, returns an optional if the file does not exist or is invalid
-if (auto trajectory = choreo::LoadTrajectory("myTrajectory")) {
-    // Do something with the trajectory
-    drive.followTrajectory(trajectory.value()).schedule();
-} else {
-    // If the trajectory is not found, ChoreoLib already prints to DriverStation
-}
-```
+=== "Java"
```

Comment:
Can I have a before and after screenshot for this page?

Reply:
Before:
![image](https://github.com/user-attachments/assets/5ecac0e5-933d-4d45-a717-0b3a7f58c0aa)
After:
![image](https://github.com/user-attachments/assets/52475f26-1bd8-4b91-8a85-a501e9289be5)

Reply:
Epic thats way better thanks!